deployment/terraform/website.tf

```diff
@@ -0,0 +1,44 @@
+#
+# ECS Resources
+#
+
+# Template for container definition, allows us to inject environment
+data "template_file" "ecs_website_task" {
+  template = "${file("${path.module}/task-definitions/site.json")}"
+
+  vars {
+    service_image = "${var.aws_account_id}.dkr.ecr.${var.aws_region}.amazonaws.com/gtsite-service:${var.image_version}"
+    static_image  = "${var.aws_account_id}.dkr.ecr.${var.aws_region}.amazonaws.com/gtsite-nginx:${var.image_version}"
+  }
+}
+
+# Allows resource sharing among multiple containers
+resource "aws_ecs_task_definition" "website" {
+  family                = "${var.environment}Website"
+  container_definitions = "${data.template_file.ecs_website_task.rendered}"
+}
+
+module "website_ecs_service" {
```

Comment:
Just so that you're aware, the [commit](https://github.com/azavea/collab-for-children/commit/9e012dd19efaf0c8664572739eafc9499b60929c) for this in C4C includes the associated CloudWatch Alarm resources that could be used to facilitate ECS service autoscaling. I don't think that's important here, but it could be useful in other settings.